R2API/RecalculateStatsAPI.cs

```diff
@@ -350,9 +353,19 @@ public class StatHookEventArgs : EventArgs {
             bool ILFound = c.TryGotoNext(
                 x => x.MatchLdarg(0),
                 x => x.MatchLdloc(out locOrigCrit),
-                x => x.MatchCallOrCallvirt(typeof(CharacterBody).GetPropertySetter(nameof(CharacterBody.crit))));
+                x => x.MatchCallOrCallvirt(typeof(CharacterBody).GetPropertySetter(nameof(CharacterBody.crit)))
+            ) && c.TryGotoPrev( MoveType.After,
+                x => x.MatchLdloc(out _),
+                x => x.MatchConvR4(),
+                x => x.MatchMul(),
+                x => x.MatchAdd(),
+                x => x.MatchCallOrCallvirt(typeof(CharacterBody).GetPropertySetter(nameof(CharacterBody.critMultiplier)))
```

Comment:
Seems like a very long chain, this can't be made shorter?

Reply:
Remnant of an older version where I grabbed the local index,whoops.
It's the first call to the setter so it can just be reduced to a MatchCallorCallvirt without affecting anything.